packages/@okta/vuepress-site/code/oie/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Identity Engine SDKs & Samples
+---
+
+# Identity Engine SDKs & Samples
+
+<ApiLifecycle access="ie" />
+
+## What's new
+
+[Identity Engine](/docs/concepts/ie-intro/) introduces new identity management capabilities with greater flexibility and customization. To take advantage of these new features and for a better development experience, use the Identity Engine SDKs to manage authentication in your apps. These new SDKs are recommended for orgs using Identity Engine.
+
+Experiment with sample apps that showcase Identity Engine using the following [authentication approaches](/docs/concepts/redirect-vs-embedded/):
+
+* **Redirect**: These sample apps demonstrate how to redirect users to an Okta-hosted sign-in page, and then receive users redirected back from Okta after users sign in. This approach is recommended for most developers, as it is easier to build and maintain.
+
+* **Embedded**: These sample apps demonstrate how to embed the Okta Sign-In Widget in an app as a package dependency.
+
+Learn how to implement each approach with these Identity Engine SDKs and sample apps in the [Redirect authentication guide](/docs/guides/sampleapp-oie-redirectauth/) and [Embedded authentication guide](/docs/guides/oie-embedded-sdk-start-with-use-case/).
+
+Orgs using Identity Engine can also continue to use current SDKs and tools. You can find those SDKs for your stack from the [Languages & SDKs overview](/code/).
+
+## Repository
+
+### Mobile
+
+##### Android
+
+<!--* [Android Identity Engine SDK](https://github.com/okta/okta-idx-android)-->
+* [Android redirect sample app](https://github.com/okta/samples-android) &mdash; see [Browser Sign In](https://github.com/okta/samples-android/tree/master/browser-sign-in) for redirect configuration
```

Comment:
global comment - this should be "redirect authentication sample app". We should be referring to redirect authentication just like we do for embedded authentication. Thanks!

Reply:
global update done. thanks!